### PR TITLE
script + migration to 5.0

### PR DESCRIPTION
**Pull request by slavagithub**

### CODE REVIEW

File: tests/demo-util-lib.sh [hunk 1]
```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-
-function check_text_in_log {
-  local log_file=$1
-    EXPECTED_TEXT=$2
-    EXPECTED_COUNT=${3:-1}
-    RESULT=PASSED
```
Comment (olukas) [inline tests/demo-util-lib.sh:7]: Seems as leftover.

File: tests/demo-util-lib.sh [hunk 2]
```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-
-function check_text_in_log {
-  local log_file=$1
```
Comment (olukas) [inline tests/demo-util-lib.sh:4]: wrong alignment

File: tests/run-tests.sh [hunk 3]
```diff
@@ -1,52 +0,0 @@
...
-cd ..
-declare -r DEMO_ROOT=$(pwd)
-
-#pack the whole project if packaging is not disabled (-p option)
-if ${PACK_JARS}
-then
-  echo "Packaging all demo projects"
-  cd ${DEMO_ROOT}
-  mvn clean package -U -B -DskipTests=true
-echo "All artifacts have been successfully built"
```
Comment (olukas) [inline tests/run-tests.sh:39]: wrong alignment

**Review by olukas — COMMENTED** [review]
Mostly minor comments.

File: flight-telemetry/src/main/java/com/hazelcast/jet/demo/FlightDataSource.java [hunk 4]
```diff
@@ -35,7 +35,7 @@
      * See <a href="https://www.adsbexchange.com/data/">ADS-B Exchange</a> for how to
      * obtain an API key.
      */
-    protected static final String API_AUTHENTICATION_KEY = "ca6a78e0d5mshecbf2798e6ba825p10879cjsnf2cc03c2e2de";
```
[the remark this answers is not in the document]
Comment (slavagithub) [reply flight-telemetry/src/main/java/com/hazelcast/jet/demo/FlightDataSource.java:38]: reverted to YOUR_KEY

File: h2o-breast-cancer-classification/log.txt [hunk 5]
```diff
@@ -1,183 +0,0 @@
-[INFO] Scanning for projects...
```
[the remark this answers is not in the document]
Comment (slavagithub) [reply h2o-breast-cancer-classification/log.txt:1]: 100% Right! Added to ignored

File: flight-telemetry/src/main/java/com/hazelcast/jet/demo/FlightTelemetry.java [hunk 6]
```diff
@@ -164,20 +163,16 @@ public static void main(String[] args) {
...
         } finally {
-            Jet.bootstrappedInstance().shutdown();
+            Jet.shutdownAll();
         }
     }
 
     private static JetInstance getJetInstance() {
         String bootstrap = System.getProperty("bootstrap");
         if (bootstrap != null && bootstrap.equals("true")) {
-            Config config = new Config();
```
[the remark this answers is not in the document]
Comment (slavagithub) [reply flight-telemetry/src/main/java/com/hazelcast/jet/demo/FlightTelemetry.java:174]: Exactly! This was this issue with the name in the config. Removed

File: pom.xml [hunk 7]
```diff
@@ -4,27 +4,29 @@
...
-    <artifactId>hazelcast-platform-demos</artifactId>
-    <version>5.0</version>
+    <groupId>com.hazelcast.jet.demos</groupId>
+    <artifactId>hazelcast-jet-demos</artifactId>
+    <version>4.3</version>
     <packaging>pom</packaging>
     <name>Hazelcast Jet Demo Applications</name>
 
     <modules>
+        <module>bitcoin-death-cross</module>
```
[the remark this answers is not in the document]
Comment (slavagithub) [reply pom.xml:14]: Let's check with him

File: tests/demo-util-lib.sh (hunk 1 above)
Comment (slavagithub) [reply tests/demo-util-lib.sh:7]: U R right, removed - played with test results

File: tests/demo-util-lib.sh (hunk 2 above)
Comment (slavagithub) [reply tests/demo-util-lib.sh:4]: Fixed!

File: tests/run-tests.sh (hunk 3 above)
Comment (slavagithub) [reply tests/run-tests.sh:39]: Fixed!

**Pull request closed without merging**